springfield/firefox/templates/firefox/features/tips/tips.html

```diff
@@ -6,6 +6,8 @@
 
 {% extends "base-protocol.html" %}
 
+{% set params = '?utm_source=www.firefox.com&utm_medium=referral&utm_campaign=firefox-features' %}
```

Comment:
(non-blocking) There's a few cards on this page linking to the blog that are missing their params.